content/v1.1.x/docs/serverless-workflows/troubleshooting.md

```diff
@@ -13,11 +13,12 @@ This document provides solutions to common problems encountered with serverless
 
 1. [Workflow Errors](#workflow-errors)
 2. [Configuration Problems](#configuration-problems)
-3. [Performance Issues](#performance-issues)
-4. [Error Messages](#error-messages)
-5. [Network Problems](#network-problems)
-6. [Common Scenarios](#common-scenarios)
-7. [Contact Support](#contact-support)
+3. [HTTP Errors](#http-errors)
+4. [Performance Issues](#performance-issues)
+5. [Error Messages](#error-messages)
+6. [Network Problems](#network-problems)
+7. [Common Scenarios](#common-scenarios)
+8. [Contact Support](#contact-support)
```

Comment:
same as before

Reply:
I tried following existing structure and i think it is better to make it visible

Reply:
fixed